src/test/java/tests/AlertsTest.java

```diff
@@ -0,0 +1,37 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pompages.AlertsPage;
+
+public class AlertsTest extends BaseTest {
+    @Test
+    public void alertJSTest() {
+        AlertsPage alertsPage = new AlertsPage(driver);
+        alertsPage.openAlertPage();
+        alertsPage.clickAlert1();
+        Assert.assertEquals("I am a JS Alert", alertsPage.getTextAlert(), "there isn't such  phrase");
+        alertsPage.closeAlert1();
+    }
+
+    @Test
+    public void alertJSConfirmTest() {
+        AlertsPage alertsPage = new AlertsPage(driver);
+        alertsPage.openAlertPage();
+        alertsPage.clickAlert2();
+        Assert.assertEquals("I am a JS Confirm", alertsPage.getTextAlert(), "there isn't such  phrase");
+        alertsPage.closeAlert2();
```

Comment:
Let's think how to close alert before assertion because if assertion happens then alert won't be closed